viz_scripts/scaffolding.py

```diff
@@ -108,6 +118,28 @@ def expand_userinputs(labeled_ct):
     disp.display(expanded_ct.head())
     return expanded_ct
 
+def expand_inferredlabels(labeled_inferred_ct):
+    if len(labeled_inferred_ct) == 0:
+        return labeled_inferred_ct
+
+    def _select_max_label(row):
```

Comment:
I am merging this for now, but note that there is a similar function in the server when we handle metrics to include trips with confidence > 90% as "confirmed". We do not and should not need to reinvent code. 

https://github.com/e-mission/e-mission-server/blob/52adee205f686d87e167bd4b1d166098938870c6/emission/analysis/result/metrics/time_grouping.py#L134